src/App.tsx

```diff
@@ -1,16 +1,17 @@
 import LoginPage from './features/auth/LoginPage/LoginPage';
-// import Button from './components/Button/Button';
 import DashboardPage from './features/dashboard/pages/DashboardPage';
-import FormRepeater from './features/home/FormRepeater/FormRepeater';
+import FormPage from './features/home/pages/FormPage';
 import { Switch, Route } from 'react-router-dom';
 
+// import Button from './components/Button/Button';
+
 function App() {
   return (
     <div>
       <Switch>
         <Route path="/" exact component={LoginPage} />
         <Route path="/dashboard" component={DashboardPage} />
-        <Route path="/form" component={FormRepeater} />
+        <Route path="/form" component={FormPage} />
```

Comment:
Let's move this components of routes in `src/routes.ts` file as additional param, like:
```javascript
import { LoginPage } from 'features/auth';

export const routes = [
  {
    path: '/',
    text: 'login',
    exact: true,
    component: LoginPage
  },
    ...
];
```

Also you need to fix your children with .map() of routes, like:
```javascript
// Move this in `types/common.ts`
type RouteType = typeof Route;
interface RouteProps extends RouteType {
  exact?: boolean;
};

import { RouteProps } from 'types/common';

const renderRoute = (params: RouteProps) => <Route {...params} />;

return (<Switch>{routes.map(renderRoute)}</Switch>);
```